cmd/run.go

```diff
@@ -51,6 +63,7 @@ kn func run
 			"You may provide this flag multiple times for setting multiple environment variables. "+
 			"To unset, specify the environment variable name followed by a \"-\" (e.g., NAME-).")
 	cmd.Flags().StringP("path", "p", cwd(), "Path to the project directory (Env: $FUNC_PATH)")
+	cmd.Flags().BoolP("build", "b", false, "Build the function only if the function has not been built before")
```

Comment:
```suggestion
	cmd.Flags().BoolP("build", "b", false, "Build the Function if it has not already been built")
```

Reply:
Since some of these commands have quite a few options, we have been bundling them up into a "config" object, and doing all the err checks, nil checks, and validation there.  That way, when it's time to actually use the values, we can just refer to `config.X`.  This seems to have cleaned things up a lot, especially in these more complex commands.

Therefore, it might be nice to add a `.Build` boolean member to the RunConfig object, populated when `newRunConfig(cmd)` is called on line 76